Formula/satysfi.rb

```diff
@@ -0,0 +1,29 @@
+class Satysfi < Formula
+  desc "Statically-typed, functional typesetting system"
+  homepage "https://github.com/gfngfn/SATySFi"
+  url "https://github.com/gfngfn/SATySFi/archive/v0.0.2.tar.gz"
+  sha256 "91b98654a99d8d13028d4c7334efa9d8cc792949b9ad1be5ec8b4cbacfaea732"
+
+  depends_on "opam" => :build
+  depends_on :x11 => :build
```

Comment:
Can probably avoid this using [this sort of silly dance](https://github.com/Homebrew/homebrew-core/blob/71576cecdc8bf61550134e56ce837e6fc27dea4a/Formula/docker-machine-driver-xhyve.rb#L55-L56)?

Reply:
OK, I'll give it a try.